### PR TITLE
Pull Request

### PR DESCRIPTION
**Pull request by ReetikaSiddanatm**

### CODE REVIEW

File: project1/application.py [hunk 1]
```diff
@@ -51,13 +53,25 @@ def authenticate():
     Registartion.query.all()
```
Comment (rehana-ms) [inline project1/application.py:53]: can you explain the  use of this line?

Comment (ReetikaSiddanatm) [reply project1/application.py:53]: I had written this line to check  the table data is been retrieved. Coming to this command it returns the primary keys of the table

Comment (rehana-ms) [reply project1/application.py:53]: when it returns dont you need to store it anywhere?

**Comment by ReetikaSiddanatm** [comment]
Madam at first I stored and checked it after while submitting I deleted the
lines

**Comment by ReetikaSiddanatm** [comment]
Moreover there is no use of storing it.

File: project1/application.py (hunk 1 above)
Comment (rehana-ms) [reply project1/application.py:53]: ok

**Pull request merged**